src/bendsave/src/main.rs

```diff
@@ -0,0 +1,113 @@
+// Copyright 2023 Databend Cloud
+//
+// Licensed under the Elastic License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.elastic.co/licensing/elastic-license
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use anyhow::Result;
+use clap::Parser;
+use clap::Subcommand;
+use databend_bendsave::backup;
+use databend_bendsave::restore;
+use logforth::append;
+use logforth::filter::EnvFilter;
+use logforth::Dispatch;
+use logforth::Logger;
+
+#[derive(Parser)]
+#[command(name = "bendsave")]
+#[command(about = "Databend backup and restore tool", long_about = None)]
+struct Cli {
```

Comment:
Can we add backup/restore subcommand in databend-query binary ?

Thus, we don't need to add an extra binary in release tars.

Reply:
Possibly. But databend-bendsave will be released under an elastic License and will require a license key to run, so I assume it will need to be used as a separate CLI.